client/src/sim.rs

```diff
@@ -6,15 +6,21 @@ use tracing::{debug, error, trace};
 
 use crate::{net, prediction::PredictedMotion, Net};
 use common::{
+    collision::BoundingBox,
+    force::{ForceParams, GravityMethod},
     graph::{Graph, NodeId},
     math,
-    node::{DualGraph, Node},
+    node::{Chunk, DualGraph, Node},
     proto::{self, Character, Command, Component, Position},
     sanitize_motion_input,
+    world::Material,
     worldgen::NodeState,
     Chunks, EntityId, GraphEntities, Step,
 };
 
+const CHARACTER_RADIUS: f64 = 0.10_f64;
+const CHARACTER_SLOWDOWN_FACTOR: f32 = 6_f32;
```

Comment:
CHARACTER_SLOWDOWN_FACTOR muzzles the client's ability to move from movement inputs, while keeping it's ability to move from persistent velocity. A value of 6 means persistent velocity is 6 times more dominant in terms of determining which way the player moves.

Reply:
Ah, that explains it. It may be easier to understand the code if the character speed from movement controls is configured entirely separately from the maximum speed from persistent velocity, but I expect that the final design will depend on whatever changes are made to server/client communication.